validator/webui/@polymer/ampproject-toolbar/ampproject-toolbar.html

```diff
@@ -74,26 +74,11 @@
            target="_blank">PRIVACY</a>
         &nbsp;&nbsp;&nbsp;
         <a href="https://chrome.google.com/webstore/detail/amp-validator/nmoffdblmcmgeicmolmhobpoocbbmknc"
-           id="install-extension" onclick="return false;" target="_blank">CHROME EXTENSION</a>
+           target="_blank">CHROME EXTENSION</a>
         &nbsp;&nbsp;&nbsp;
         <a href="https://www.npmjs.com/package/amphtml-validator"
            target="_blank">NPM</a>
       </div>
     </paper-toolbar>
   </template>
```

Comment:
Can you delete the imports for polymer and iron-flex-layout above? They don't seem to be referenced in this file, but I dunno if they cause some necessary magic.